worker/test/src/RTC/RTCP/TestFeedbackRtpTransport.cpp

```diff
@@ -571,4 +571,223 @@ SCENARIO("RTCP Feeback RTP transport", "[parser][rtcp][feedback-rtp][transport]"
+                                0x26, 0x9E, 0x8E, 0x50, 0x03, 0xAE, 0x00, 0x06,
+                                0x0C, 0x7F, 0x35, 0xA1, 0x20, 0x06, 0x9C, 0x34,
+                                0x24, 0x14, 0x10, 0x34
+                        }
+                },
+                {
+                        .baseTimeRaw = 821523,
+                        .baseTimeMs = 1126319296,
+                        .baseSequence = 10,
+                        .packetStatusCount = 7,
+                        .deltas = std::vector<int16_t>{25, 2, 2, 3, 1, 1, 3},
+                        .buffer = std::vector<uint8_t> {
+                                0xAF, 0xCD, 0x00, 0x07, 0xFA, 0x17, 0xFA, 0x17,
+                                0x00, 0x00, 0x04, 0xD2, 0x00, 0x0A, 0x00, 0x07,
+                                0x0C, 0x89, 0x13, 0x00, 0x20, 0x07, 0x64, 0x08,
+                                0x08, 0x0C, 0x04, 0x04, 0x0C, 0x00, 0x00, 0x03
+                        }
+                },
+                {
+                        .baseTimeRaw = 821524,
+                        .baseTimeMs = 1126319360,
+                        .baseSequence = 17,
+                        .packetStatusCount = 2,
+                        .deltas = std::vector<int16_t>{44, 18},
+                        .buffer = std::vector<uint8_t> {
+                                0x8F, 0xCD, 0x00, 0x05, 0xFA, 0x17, 0xFA, 0x17,
+                                0x08, 0xEB, 0x06, 0xD7, 0x00, 0x11, 0x00, 0x02,
+                                0x0C, 0x89, 0x14, 0x01, 0x20, 0x02, 0xB0, 0x48
+                        }
+                },
+                {
+                        .baseTimeRaw = 821524,
+                        .baseTimeMs = 1126319360,
+                        .baseSequence = 17,
+                        .packetStatusCount = 1,
+                        .deltas = std::vector<int16_t>{62},
+                        .buffer = std::vector<uint8_t> {
+                                0xAF, 0xCD, 0x00, 0x05, 0xFA, 0x17, 0xFA, 0x17,
+                                0x20, 0x92, 0x5E, 0xB7, 0x00, 0x11, 0x00, 0x01,
+                                0x0C, 0x89, 0x14, 0x00, 0x20, 0x01, 0xF8, 0x01
+                        }
+                },
+                {
+                        .baseTimeRaw = 821526,
+                        .baseTimeMs = 1126319488,
+                        .baseSequence = 19,
+                        .packetStatusCount = 4,
+                        .deltas = std::vector<int16_t>{4, 0, 4, 0},
+                        .buffer = std::vector<uint8_t> {
+                                0xAF, 0xCD, 0x00, 0x06, 0xFA, 0x17, 0xFA, 0x17,
+                                0x08, 0xEB, 0x06, 0xD7, 0x00, 0x13, 0x00, 0x04,
+                                0x0C, 0x89, 0x16, 0x02, 0x20, 0x04, 0x10, 0x00,
+                                0x10, 0x00, 0x00, 0x02
+                        }
+                }
+        };
+
+        for (const auto &packetMeta: feedbackPacketsMeta) {
```

Comment:
Please run `make lint` in worker/ directory. It should complain. `make format` will fix it.